src/components/AvatarWithImagePicker.js

```diff
@@ -154,6 +166,13 @@ class AvatarWithImagePicker extends React.Component {
      * @param {Object} image
      */
     showAvatarCropModal(image) {
+        if (!this.isValidExtension(image)) {
+            this.showErrorModal(
+                this.props.translate('avatarWithImagePicker.imageUploadFailed'),
+                `${this.props.translate('avatarWithImagePicker.notAllowedExtension')} ${CONST.AVATAR_ALLOWED_EXTENSIONS.join(', ')}.`,
```

Comment:
```suggestion
                `${this.props.translate('avatarWithImagePicker.notAllowedExtension', {  allowedExtensions: CONST.AVATAR_ALLOWED_EXTENSIONS })}`,
```

In inline with the translation message comment, this would be